airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -144,6 +144,14 @@ def get_json_schema(self) -> Mapping[str, Any]:
             }
         )
 
+        if "cohort_spec" not in self.config and "date" not in self.config["dimensions"]:
```

Comment:
can you add a comment on this block? It's not obvious why "cohort_spec" is a special case

Reply:
added comment, cohorts  doesn't support startDate and endDate